test/cypress/e2e/moderationCreateHealthcareProfessional.cy.ts

```diff
@@ -29,28 +26,20 @@ describe('Moderation create healthcare professional button', () => {
 
 describe('Moderation create healthcare professional form', () => {
     context('Landscape mode', () => {
-        before(() => {
-            cy.intercept('POST', '**/', req => {
-                if (req.body.operationName === 'Query') {
+        beforeEach(() => {
+            cy.intercept('POST', '**', req => {
```

Comment:
```suggestion
            cy.intercept('POST', '**/', req => {
```

You forgot the slash